src/rev_ai/apiclient.py

```diff
@@ -142,6 +142,23 @@ def get_transcript_text(self, id_):
 
         return response.text
 
+    def get_transcript_json(self, id_):
+        """Get the transcript of a specific job as json
+
+        :param id_: id of job to be requested
+        :returns: transcript data as json
+        :raises: HTTPError
+        """
+        if not id_:
+            raise ValueError('id_ must be provided')
+
+        url = urljoin(self.base_url, 'jobs/{}/transcript'.format(id_))
+        response = self.session.get(
+            url, headers={'Accept': 'application/{}+json'.format('vnd.rev.transcript.v1.0')})
```

Comment:
do you want to store the value `vnd.rev.transcript.v1.0` in a class variable

Reply:
Seems reasonable